de.fu_berlin.inf.dpp.core/src/de/fu_berlin/inf/dpp/context/ContainerContext.java

```diff
@@ -31,14 +31,14 @@
 import java.io.File;
 import java.util.List;
 import org.apache.log4j.Logger;
-import org.picocontainer.ComponentMonitor;
-import org.picocontainer.MutablePicoContainer;
-import org.picocontainer.PicoBuilder;
-import org.picocontainer.PicoContainer;
-import org.picocontainer.injectors.AnnotatedFieldInjection;
-import org.picocontainer.injectors.CompositeInjection;
-import org.picocontainer.injectors.ConstructorInjection;
-import org.picocontainer.injectors.Reinjector;
+import saros.org.picocontainer.ComponentMonitor;
```

Comment:
The convention is to reverse the domain so it is: org.saros.* and not saros.org

Reply:
My guess would be that this package name was created by just prepending `saros` to the default picocontainer package name (`org.picocontainer`). But I am not sure what a good package name/path would be.